.github/tools/matrixcheker/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/exp/slices"
+	"gopkg.in/yaml.v2"
+)
+
+const workflowFile = "./.github/workflows/tests.yaml"
+
+const jobName = "warehouse-integration"
+
+var IgnorePackages = []string{
+	"warehouse/integrations/deltalake",
+	"warehouse/integrations/postgres-legacy",
+	"warehouse/integrations/manager",
+	"warehouse/integrations/middleware",
+	"warehouse/integrations/testhelper",
+}
```

Comment:
Can we add testdata as well in here?

```suggestion
var IgnorePackages = []string{
	"warehouse/integrations/deltalake",
	"warehouse/integrations/postgres-legacy",
	"warehouse/integrations/manager",
	"warehouse/integrations/middleware",
	"warehouse/integrations/testhelper",
	"warehouse/integrations/testdata",
}
```